core/vnl/vnl_vector.hxx

```diff
@@ -105,7 +105,9 @@ vnl_vector<T>::vnl_vector (size_t len, size_t n, T const values[])
 {
   vnl_vector_alloc_blah(len);
   // If user specified values, initialize first n elements with values
-  const size_t copy_num{std::min(len,n) };
+  // n.b assignment is used over universal initialization for avoid a
+  // gcc 4.8.5 ICE
+  const size_t copy_num = std::min(len,n);
```

Comment:
I finished building vxl in the docker container you provided at 2:06pm :) 2 min before this arrived.    I made the exact same change!

Hans